src/ansys/hps/client/client.py

```diff
@@ -190,6 +191,13 @@ def __init__(
                 verify=self.verify,
             )
             self.access_token = tokens["access_token"]
+
+            try:
+                parsed_jwt = jwt.decode(self.access_token, options={"verify_signature": False})
+                self.username = parsed_jwt.get("preferred_username", self.username)
+            except:
+                log.warning("Could not retrieve username from access token")
```

Comment:
```suggestion
                log.warning("Could not retrieve username from access token.")
```
Messages should have concluding punctuation, which is virtually always a period.

Reply:
*Added punctuation.